packages/edit-post/README.md

```diff
@@ -181,30 +182,29 @@ function MyButtonMoreMenuItem() {
 			icon: moreIcon,
 			onClick: onButtonClick,
 		},
-		__( 'My button title' )
+		\_\_( 'My button title' )
 	);
 }
-```
-
-```jsx
-// Using ESNext syntax
-import { __ } from '@wordpress/i18n';
-import { PluginMoreMenuItem } from '@wordpress/edit-post';
-import { more } from '@wordpress/icons';
-
-function onButtonClick() {
-	alert( 'Button clicked.' );
-}
 
-const MyButtonMoreMenuItem = () => (
-	<PluginMoreMenuItem
-		icon={ more }
-		onClick={ onButtonClick }
-	>
-		{ __( 'My button title' ) }
-	</PluginMoreMenuItem>
-);
-```
```

Comment:
It removes closing "```"

Reply:
I'm really confused why this is happening. The description has the trailing ```` ``` ```` in it when I inspect it, I have no idea why it's being removed.

To make matters worse, when I arbitrarily append a random string like `asdf` to the end of the description, the ```` ``` ```` suddenly appears with `asdf` appended. It's very odd.